lib/api/src/grpc/conversions.rs

```diff
@@ -354,6 +504,15 @@ impl TryFrom<GeoRadius> for segment::types::GeoRadius {
     }
 }
 
+impl From<segment::types::GeoRadius> for GeoRadius {
+    fn from(value: segment::types::GeoRadius) -> Self {
+        Self {
+            center: Some(value.center.into()),
+            radius: value.radius as f32, // TODO lossy ok?
```

Comment:
do we need to document this loss of precision from f64 to f32 somewhere?

Reply:
Is it possible to declare the with the same type (e.g. both f64 or both f32)? Or there are some additional limitations?

Reply:
I believe we use an `f32` and not a `f64` here because it corresponds to the `float` in the proto. definition.
https://developers.google.com/protocol-buffers/docs/proto3

Reply:
Yeah, but they also have `double` there which is f64 as far as I understand.

Reply:
Anyway this can be handled in a separate issue, maybe @generall also has some thoughts on this

Reply:
I agree, I'd rather not change this API contract in the middle of this giant PR :+1: